### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by grayTurtle01**
Added arrow-function and ternary operator

### 2. Description
(Write description of your changes here)

### 3. Fill in checklist by marking [x]

- [x] I've read the CONTRIBUTING.md
- [ ] I was assigned to this issue
- [x] My code is formatted using required linting
- [x] I've run my code locally and checked it works

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Each file contains:
 | **Variables**                              |                                                                             |
 | Create variables                           | [creating-variables.js](2-variables/creating-variables.js)                  |
 | JavaScript Numbers                         | [numbers.js](2-variables/numbers.js)                                        |
-| Booleans in JavaScript                     | [booleans.js](2-variables/booleans.js)                                        |
+| Booleans in JavaScript                     | [booleans.js](2-variables/booleans.js)                                      |
```
Comment (sabderemane) [inline README.md:26]: What did you changed here? Normally no need for a change

File: README.md [hunk 2]
```diff
@@ -37,6 +37,7 @@ Each file contains:
 | Operators in JavaScript                    | [operators.js](5-operators/operators.js )                                   |
 | **Conditions**                             |                                                                             |
 | Conditionals                               | [conditions.js](6-conditions/conditions.js )                                |
+| Ternary Operator                           | [ternay-operator.js](6-conditions/ternary-operator.js)                      |
```
Comment (sabderemane) [inline README.md:40]:
```suggestion
| Ternary Operator                           | [ternary-operator.js](6-conditions/ternary-operator.js)                      | 
```

File: README.md [hunk 3]
```diff
@@ -45,5 +46,6 @@ Each file contains:
 | **Functions**                              |                                                                             |
 | Functions in JavaScript                    | [functions.js](8-functions/functions.js )                                   |
 | JavaScript's common built-in functions     | [built-in-functions.js](9-common-built-in-functions/built-in-functions.js ) |
+| Arrow Functions                            | [arrow-functions.js](8-functions/arrow-function.js)                         |
```
Comment (sabderemane) [inline README.md:49]:
```suggestion
| Arrow Functions                            | [arrow-function.js](8-functions/arrow-function.js)                         | 
```

**Review by sabderemane — CHANGES_REQUESTED** [review]

**Comment by sabderemane** [comment]
Thank you @grayTurtle01 for updating the README ! I suggest some changes :)

File: README.md (hunk 1 above)
Comment (grayTurtle01) [reply README.md:26]: I removed a empty space, for a consistent format.

**Comment by grayTurtle01** [comment]
Hey @sabderemane Thaks for the suggestions

File: README.md (hunk 2 above)
Comment (grayTurtle01) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (grayTurtle01) [reply README.md:49]: Done

**Review by sabderemane — APPROVED** [review]

**Pull request merged**